VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
It seems OK to me to shift to `*.VC.db` and `*.VC.opendb`, I do indeed see that the file always has the `VC` suffix.

I'd like to see if we can find anyone from the VC++ team to help. I can try to put out feelers, but I'm not sure I have the contacts :). Worst case, I'll just email Satya :trollface: